### ISSUE TITLE
Message from onClick is getting dispatched twice on click

### ISSUE THREAD

**Issue by hrb90** [mdgriffith/style-elements#38]
See [this Ellie](https://ellie-app.com/3HF8fsypvDRa1/0) for a SSCCE. Moving the onClick event to an Html div and using the onClick from Html.Events works; see [this Ellie](https://ellie-app.com/3HF8fsypvDRa1/1).

**Comment by opsb** [comment]
The duplicate update disappears when `verticalCenter` is removed. `verticalCenter` introduces 2 intermediate divs, if you look at the second you'll see that it has the additional click listener attached.

**Comment by hrb90** [comment]
I played around in the Ellie and it looks like all of the HAlign or VAlign attributes introduce two click listeners.

**Comment by AionDev** [comment]
i'm using `alignRight` in a row, and i get the same 2 clicks if i attach the event to the row:
```
row None
    [ alignRight
    , onClick <| Messages.ToggleBucket messageId bucketType
    ]
    [ el BucketName [] <|
        text "why 2 clicks instead of one?"
    ]
```
I need to align something to the right, and i don't know how to do it without encountering this bug..

**Comment by mdgriffith** [comment]
can you just wrap it in an el and attach the event to that el?

```elm
el [ onClick <| Messages.ToggleBucket messageId bucketType ] <|
    row None
        [ alignRight
        ]
        [ el BucketName [] <|
             text "why 2 clicks instead of one?"
        ]
```

**Comment by AionDev** [comment]
that nested `el` had 4 other siblings, but i aded `onClick` to each of them, even if it's more code it works as expected. 😄 Thanks, look forward to see this one fixed 👍

**Comment by mdgriffith** [comment]
OK cool.  This will be fixed in the next release 😄

**Comment by mdgriffith** [comment]
This was fixed in `v4.0` I believe, just forgot to close this issue.

Thanks!
